### PR TITLE
[release/v2.22] Change logs for patch release 2.22.13

### PR DESCRIPTION
**Pull request by akash-gautam**
**What this PR does / why we need it**:
Change log for the upcoming patch release.

**Which issue(s) this PR fixes**:

Fixes #

**What type of PR is this?**
/kind documentation

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change? Then add your Release Note here**:

```release-note
NONE
```

**Documentation**:

```documentation
NONE
```

### CODE REVIEW

**Comment by akash-gautam** [comment]
cc @embik @archups

File: docs/changelogs/CHANGELOG-2.22.md [hunk 1]
```diff
@@ -13,6 +13,14 @@
...
 - [v2.22.11](#v22211)
 - [v2.22.12](#v22212)
+- [v2.22.13](#v22213)
+
+## [v2.22.13](https://github.com/kubermatic/kubermatic/releases/tag/v2.22.13)
+
+### Updates
+
+- Update metering to v1.0.6, fixing an error when a custom CA bundle is used ([#13011](https://github.com/kubermatic/kubermatic/pull/13011))
+- Update operating-system-manager (OSM) to [v1.2.4](https://github.com/kubermatic/operating-system-manager/releases/tag/v1.2.4)  * This includes a fix for Flatcar stable channel (version 3815.2.0) failing to provision as new nodes ([#13084](https://github.com/kubermatic/kubermatic/pull/13084))
```
Comment (xrstf) [inline docs/changelogs/CHANGELOG-2.22.md]: Please fix formatting, the second part could be in parenthesis or something. In the 2.23 PR you made it differently already.

**Review by xrstf — APPROVED** [review]
/approve

**Pull request merged**